OSVR-Unity/Assets/OSVRUnity/src/VRHead.cs

```diff
@@ -63,12 +62,12 @@ void Start()
             {
                 Init();
                 CatalogEyes();
-                _distortionEffect = GetComponent<OsvrDistortion>();
+/*
```

Comment:
Is this commented out code going to be re-enabled in the future, or replaced with something else? If replaced with something else, it's probably better to just remove it and add a TODO note.